src/js/actions/menu.js

```diff
@@ -428,7 +429,7 @@ define(function (require, exports) {
             actionModuleName = actionNameParts[0],
             actionName = actionNameParts[1],
             actionNameThrottled = actionName + "Throttled",
-            actionThrottled = this.flux.actions[actionModuleName][actionNameThrottled];
+            actionThrottled = objUtil.getPath(this.flux.actions, actionModuleName)[actionNameThrottled];
```

Comment:
Yes, ideally. But as you said, it's not exposed. I think exposing FluxController will allow us to shoot ourselves in the foot easier, and not having it readily available makes us limit calls like this, which is a good thing.